tools/build_hooks/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
As the only python file in this directory should be moved to a different directory, could you delete this file?